ManagedWinapi/Windows/SystemListView.cs

```diff
@@ -207,6 +291,16 @@ internal SystemListViewItem(SystemWindow sw, int index, string title, uint state
         public int Image { get { return image; } }
 
         /// <summary>
+        /// The appearance of a selected item depends on whether it has the focus and also on the system colors used for selection.
+        /// </summary>
+        public bool IsSelected { get { return (state & (uint)ListViewItemState.LVIS_SELECTED) != 0; } }

```

Comment:
I think it is more .NET-like to have a "Selected" property with a getter and a setter (instead of an IsSelecteed property and a method to set it), but this solution is fine, too.